### PR TITLE
Fix streaming Audio/Video Output

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

I broke it in #9589 

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: gradio/chat_interface.py [hunk 1]
```diff
@@ -466,7 +466,7 @@ def _setup_stop_events(
                 None,
                 None,
                 None,
-                cancels=events_to_cancel,
+                cancels=events_to_cancel,  # type: ignore
```
Comment (abidlabs) [inline gradio/chat_interface.py:469]: Putting this for now so that we can satisfy typecheck and run the backend tests

**Review by abidlabs — APPROVED** [review]
Confirmed that this fixes the issue in this demo:

```py
import gradio as gr
from time import sleep

def keep_repeating(audio_file):
    for _ in range(10):
        sleep(0.5)
        yield audio_file

gr.Interface(keep_repeating,
             gr.Audio(sources=["microphone"], type="filepath"),
             gr.Audio(streaming=True, autoplay=True)
).launch()
```

Do we have an e2e test for streaming out?

**Comment by freddyaboulton** [comment]
Yes we have e2e tests for streaming!

**Comment by freddyaboulton** [comment]
Thank you @abidlabs !

**Pull request merged**